larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -501,6 +508,13 @@ StatusCode MasterAlgorithm::RunSliceReconstruction(SliceVector &sliceVector, Sli
             PANDORA_RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraApi::ProcessEvent(*m_pSliceNuWorkerInstance));
             PANDORA_RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraApi::GetCurrentPfoList(*m_pSliceNuWorkerInstance, pSliceNuPfos));
             nuSliceHypotheses.push_back(*pSliceNuPfos);
+    
+            for (const ParticleFlowObject *const pPfo : *pSliceNuPfos)
+            {
+                PandoraContentApi::ParticleFlowObject::Metadata metadata;
+                metadata.m_propertiesToAdd["SliceIndex"] = sliceCounter;
```

Comment:
So SliceIndices start at 1? That's OK, if consistent throughout all treatment. Helps to keep code clean here.

Reply:
Yes! I index from 1 here to be consistent with couts in the master instance